docusaurus.config.js

```diff
@@ -239,21 +239,26 @@ module.exports = {
           editUrl: 'https://github.com/okteto/docs/edit/main',
           breadcrumbs: false,
           sidebarPath: require.resolve('./sidebars.js'),
-          lastVersion: '1.24',
+          lastVersion: '1.25',
           versions: {
             current: {
               // aka unreleased version in development
               // Remember to also update "unreleased" redirect if changing the value!
-              label: '1.25',
-              path: '1.25',
+              label: '1.26',
+              path: '1.26',
             },
             1.22: {
```

Comment:
Is this correct? Shouldn't it be `1.25`?

Reply:
This is always one version ahead

Reply:
I refer to the key. It says `1.22` which makes me think it is not right. So, in the `versions`  object, it seems there are elements with the key `1.22`

Reply:
I see what you're saying. It's odd that this compiles just fine. When I change the key to 1.25 it has a failure.
Figuring out what's going on...